examples/CMakeLists.txt

```diff
@@ -28,7 +28,7 @@ target_link_libraries(regression
   ${Boost_LIBRARIES})
 install_targets(/bin/skylark_examples regression)
 
-add_executable(random_features random_features.cpp)
+add_executable(random_features random_features.cpp ${KISSFFT_SOURCE_FILES})
```

Comment:
This should be removed. When I started to integrate kissfft I did it with the C version (because the C++ version does not have fftr2r) The C version uses .h as interface and .c for the implementation. I had to provide the .c to the programs that were going to use kissfft. However, we implemented our own version of kissfftr2r using the C++ version which is only a .hh file (templatized).

Summarizing: This has to be removed